internal/command/worker/run.go

```diff
@@ -23,6 +26,8 @@ func newRunCommand() *cobra.Command {
 	}
 	cmd.PersistentFlags().StringVar(&bootstrapTokenRaw, "bootstrap-token", "",
 		"a bootstrap token retrieved via `orchard get bootstrap-token <service-account-name-for-workers>`")
+	cmd.PersistentFlags().StringVar(&logFilePath, "log-file", "/tmp/orchard-worker.log",
```

Comment:
I don't think everyone wants to log to a file? IMO the standard output/error should be the default, [as in Persistent Worker](https://github.com/cirruslabs/cirrus-cli/blob/master/PERSISTENT-WORKERS.md#logging), for example.

Reply:
SGTM. See [ed35e54](https://github.com/cirruslabs/orchard/pull/59/commits/ed35e549e30a2edc44f56ec07cc3ad27d57df320)